compiler/compile/OSRData.cpp

```diff
@@ -586,7 +596,11 @@ void TR_OSRCompilationData::buildDefiningMap()
       else osrCodeBlockRemoved = true;
 
       if (!osrCatchBlockRemoved && !osrCodeBlockRemoved )
-         buildFinalMap(i-1, osrMethodData->getDefiningMap(), definingMapAtOSRCatchBlocks[i], definingMapAtOSRCodeBlocks, definingMapAtPrepareForOSRCalls);
+         {
+         DefiningMap *finalMap = new DefiningMap(DefiningMapComparator(), DefiningMapAllocator(region));
```

Comment:
Doesn't this need to be a placement new - the global new won't do what you want....

Reply:
Ugh!  Thanks for catching that.

Reply:
Corrected in commit https://github.com/eclipse/omr/pull/5259/commits/bb10e07e5bb57e1b71d67a64f798c96554af378c